src/zui/components/ZUIItemCard/index.tsx

```diff
@@ -0,0 +1,107 @@
+import { FC } from 'react';
+import { Box, Typography } from '@mui/material';
+
+import ZUIAvatar, { ZUIAvatarProps } from '../ZUIAvatar';
+import ZUIButton from '../ZUIButton';
+
+type ZUIItemCardProps = {
+  avatar?: Omit<ZUIAvatarProps, 'size' | 'variant'>;
+
+  button?: {
+    /**
+     * The label of the button
+     */
+
+    label: string;
```

Comment:
```suggestion
    label: string;

```

Reply:
I'm gonna leave it for now, will do a total go-through of props on all components and it can be sorted in that part of the work :)